grader/grader.py

```diff
@@ -11,7 +11,8 @@ class Grader(BaseGrader):
 #
     def eval_solution(self, test_case) -> bool:
         """Evaluates solution against the given test case."""
-        input_data, want = test_case
+        input_data = list(test_case[0])
+        want = list(test_case[1])
```

Comment:
This isn't necessary. Original code is doing this exact same thing. The previous syntax is called unpacking.

Reply:
If I leave it as it was and break my code, then the Error message shows the same values for input and output (See image below), I think it is because we must modify the original list, and this modify test_case as well.

![image](https://user-images.githubusercontent.com/58634102/82737010-6a24bd00-9cf3-11ea-8a00-823d25eb833b.png)

Reply:
It is right, but that's not the solution. He have to create a copy of the input array before sending it to the evaluation method. Feel free to implement the copy operation in a different PR <3 

Docs: https://docs.python.org/3/library/copy.html

Reply:
Ok, Will do it that way.
Thank you!